views/add_peep.erb

```diff
@@ -0,0 +1,4 @@
+<form method="post" action="/add_peep">
+  <input type="text" name="peep" placeholder="your peep here"required>
+  <input type="submit" name="Submit">
+</form>
```

Comment:
I like the simplicity of the form, it is very user friendly

Reply:
thanks!